util/dashboard/dashboard_validate.py

```diff
@@ -38,15 +38,43 @@ def check_keys(obj, required_keys, optional_keys, err_prefix):
     'notes': ['s', "random notes"],
 }
 
+entry_required = {
+    'version': ['s', "module version"],
+    'life_stage': ['s', "life stage of module"]
+}
+entry_optional = {
+    'design_stage': ['s', "design stage of module"],
+    'verification_stage': ['s', "verification stage of module"],
+    'commit_id': ['s', "Staged commit ID"],
+    'notes': ['s', "notes"],
+}
+
 
 def validate(regs):
     if not 'name' in regs:
         log.error("Component has no name. Aborting.")
         return 1
     component = regs['name']
 
-    error = check_keys(regs, field_required, field_optional, component)
+    if not "revisions" in regs:
+        # Version1  checker
```

Comment:
Move comment above `if` statement.  Full sentence on what this does, something like: `Handle older revision format.`

Reply:
OK. remembering always comment above...